.circleci/pyscripts/generate_main.py

```diff
@@ -305,29 +242,32 @@ def combine_workflow_path(job_entry_spaces, workflow_path):
                 )
                 workflow_path = append_to_sample_workflow(workflow_path, workflow)
 
-            if build_components["build"]["build"]:
-                print("build> build:", circleCI.get_Workflow_dependency("build"))
+            if build_components["build-deploy"]:
```

Comment:
long-term, this entire collection of "ifs" should get turned into something programmatic that just reads the workflow file and iterates over it, rather than hardcoded names, but for now, this gets it working

Reply:
I agree. I didn't implement that for this PR as it would be a rather large change.